presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -322,11 +324,41 @@ private static void process(QueryRunner queryRunner, String sql, OutputFormat ou
             queryRunner.setSession(session);
         }
         catch (RuntimeException e) {
-            System.err.println("Error running command: " + e.getMessage());
-            if (queryRunner.getSession().isDebug()) {
-                e.printStackTrace();
+            System.err.println(getErrorMessage(e, queryRunner.getSession()));
+        }
+    }
+
+    private static String getErrorMessage(RuntimeException e, ClientSession session)
+    {
+        StringWriter sw = new StringWriter();
+        e.printStackTrace(new PrintWriter(sw));
+        String stackTrace = sw.toString();
+
+        StringBuilder builder = new StringBuilder();
+        if (e.getCause() instanceof java.io.EOFException) {
+            builder.append("There was a problem with a response from Presto Coordinator." + "\n");
```

Comment:
These string literals can be joined.

The phrasing here is a bit off. I'll improve it before merging.